src/OrchardCore.Modules/OrchardCore.Github/Recipes/GithubSigninSettingsStep.cs

```diff
@@ -8,15 +8,15 @@
 namespace OrchardCore.Github.Recipes
 {
     /// <summary>
-    /// This recipe step sets Microsoft Account settings.
+    /// This recipe step sets Github Account settings.
     /// </summary>
     public class GithubSigninSettingsStep : IRecipeStepHandler
     {
-        private readonly IGithubAuthenticationService _twitterLoginService;
+        private readonly IGithubAuthenticationService _githubLoginService;
 
```

Comment:
_githubAuthenticationService ? Also name of file and class should be GithubAuthenticationSettingsStep, as this name is used in settings, views etc?

Reply:
Made the changes as per your comment, I don't think there's any standard name convention across all Authentication modules (twitter, facebook etc). They all seem different.

Reply:
indeed, tried to use the name used in the docs of the provider, ex Twitter Signin, Facebook Login etc